main/ui/src/main/java/org/cryptomator/ui/addvaultwizard/CreateNewVaultLocationController.java

```diff
@@ -70,27 +74,44 @@
 		this.vaultPath = vaultPath;
 		this.vaultName = vaultName;
 		this.resourceBundle = resourceBundle;
-		this.validVaultPath = Bindings.createBooleanBinding(this::isValidVaultPath, vaultPath);
+		this.validVaultPath = Bindings.createBooleanBinding(this::isValidVaultPath, this.vaultPath);
 		this.usePresetPath = new SimpleBooleanProperty();
 		this.warningText = new SimpleStringProperty();
 	}
 
 	private boolean isValidVaultPath() {
-		return vaultPath.get() != null && Files.notExists(vaultPath.get());
+		final Path p = vaultPath.get();
+		if (p == null) {
+			warningText.set("ERROR");
```

Comment:
I agree that it probably doesn't have to be localized but I believe there is a better description than "ERROR".

Reply:
Changed in dd4c163